.github/workflows/docs.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish documentation
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Fetch all git branches
+        run: git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - run: yarn
+      - run: yarn build:docs
+      - uses: docker://malept/gha-gh-pages:1.0.2
```

Comment:
Not block, just question: why use docker image? It's unable to do via `actions` api?

Reply:
The action itself is a Docker container, so it's faster to run as a Docker container rather than an action (where it would execute the Dockerfile on every run)